training/architecture/DATA_TERMS.md

```diff
@@ -0,0 +1,57 @@
+
+Within editions, we must ensure we can adapt to errors or provide more up to date information. **Versions** are the specific releases of an edition of a dataset. If a user downloads a file, it is a specific version for example, version 2 of the 2019 edition of the Annual Count of Welsh Pets dataset.
+
+Datasets and editions use names which provide some context, whereas versions are always incrementing counts (1,2,3...)
+
+### What kinds of datasets exist?
+[This sales by product](https://www.ons.gov.uk/businessindustryandtrade/manufacturingandproductionindustry/datasets/ukmanufacturerssalesbyproductprodcom) dataset page has a 'Current' edition and the latest version is accessible through the XLSX link. Other versions can be seen in the 'Previous versions' link.
+
+[The CPI time series](https://www.ons.gov.uk/economy/inflationandpriceindices/datasets/consumerpriceindices) dataset page doesn't feature the same edition container as timeseries datasets are an exception where there is only 1 fixed edition and every release is only noted as a separate version. 
+
+In a time series dataset, each version of the dataset will contain all previous data _and_ the most recent update. In an editioned dataset, each edition may only contain the information relevant to its timeframe or view on the dataset.
+
+Some datasets are available through the Customize my Data system. These datasets are known as Filterable datasets, and follow all the same edition and versioning rules. [These datasets](https://www.ons.gov.uk/datasets/mid-year-pop-est/editions/mid-2019-april-2020-geography/versions/2) feature the edition and version information in the URL, and provide additional functionality for users to request subsets of data that they're interested in.
+
+### What is the data made up of?
+
+Take this CSV snippet as an example of some statistics:
+``` csv
+Count, Area, House Type, Pets
+1, Cardiff, Detached, Dogs
+28, Caridff, Terraced, Dogs
+5, Newport, Detached, Cats
+98, Newport, Terraced, Cats
+```
+This CSV might be the version 2 file of the Annual Count of Welsh Pets we mentioned earlier. This file contains 4 **observations**, across 3 **dimensions** each featuring 2 **options**.
+
+An **observation** is a specific statisticial value. These may be counts, percentages, indexes or any other value.
+
+A **dimension** is one of the categories of information the dataset is broken down into - the columns in this CSV: area, house type and pets.
+
+**Dimension options** are the specific values available within a dimension. For the area dimension, Cardiff and Newport are the options.
+
+An observation can be defined as a _"unique combination of dimension options"_. There can't be 2 possible values for Newport, Terraced, Cats; one of the options would have to change to result in a different observation figure.
```

Comment:
Is it worth us adding what a CMD dataset 'instance' is? I found this super beneficial when someone explained this to me